Library/Formula/github-markdown-toc.rb

```diff
@@ -0,0 +1,40 @@
+require "language/go"
+
+class GithubMarkdownToc < Formula
+  desc "Easy TOC creation for GitHub README.md (in go)"
+  homepage "https://github.com/ekalinin/github-markdown-toc.go"
+  url "https://github.com/ekalinin/github-markdown-toc.go/archive/0.5.0.tar.gz"
+  sha256 "78ea87ab18fd213e52b217d7ec0bc4eb56a8dbeb2331b82376e6a741b0121d1d"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/alecthomas/template" do
+    url "https://github.com/alecthomas/template.git",
+      :revision => "14fd436dd20c3cc65242a9f396b61bfc8a3926fc"
+  end
+
+  go_resource "github.com/alecthomas/units" do
+    url "https://github.com/alecthomas/units.git",
+      :revision => "2efee857e7cfd4f3d0138cc3cbb1b4966962b93a"
+  end
+
+  go_resource "gopkg.in/alecthomas/kingpin.v2" do
+    # v2.1.11
+    url "https://github.com/alecthomas/kingpin.git",
+      :revision => "8cccfa8eb2e3183254457fb1749b2667fbc364c7"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/ekalinin/"
+    ln_sf buildpath, buildpath/"src/github.com/ekalinin/github-markdown-toc.go"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "go", "build", "-o", "gh-md-toc", "main.go"
+    bin.install "gh-md-toc"
+  end
+
+  test do
+    system bin/"gh-md-toc", "--version"
```

Comment:
Oh, yup – since it was added as a new line after this, instead of changing the line this line-comment sequence is on, and in a separate commit, it doesn't show up on the page with this comment thread. That's one reason we prefer PRs to be squashed to a single commit per formula: makes it more obvious what the current state of the PR's code is.

Would you mind squashing this PR to a single commit? Then I think we're good to merge.